packages/redux-subspace-observable/src/observable/subspaced.js

```diff
@@ -16,15 +16,17 @@ const identity = (x) => x
 const subspaced = (mapState, namespace) => {
     const subspaceDecorator = subspace(mapState, namespace)
     return epic => (action$, store, dependencies) => {
-        const subspacedStore = subspaceDecorator(store)
+        const subspacedStore = subspaceDecorator(dependencies.store)
```

Comment:
might want some validation on whether `dependencies` is an object and it has a `store` field (i.e. they have now used our `createEpicMiddleware` wrapper) for development